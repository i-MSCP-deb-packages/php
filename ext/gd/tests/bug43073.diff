001+ (499, 401), (614, 401), (614, 374), (499, 374)
002+ (492, 363), (598, 319), (588, 294), (481, 338)
003+ (471, 330), (552, 249), (533, 230), (452, 311)
004+ (439, 308), (483, 201), (458, 191), (414, 297)
005+ (401, 300), (401, 185), (374, 185), (374, 300)
006+ (363, 306), (319, 200), (294, 210), (338, 317)
007+ (330, 327), (249, 246), (230, 265), (311, 346)
008+ (308, 359), (201, 315), (191, 340), (297, 384)
009+ (300, 398), (185, 398), (185, 425), (300, 425)
010+ (306, 435), (200, 479), (210, 504), (317, 460)
011+ (327, 468), (246, 549), (265, 568), (346, 487)
012+ (359, 490), (315, 597), (340, 607), (384, 501)
013+ (398, 499), (398, 614), (425, 614), (425, 499)
014+ (435, 492), (479, 598), (504, 588), (460, 481)
015+ (468, 471), (549, 552), (568, 533), (487, 452)
016+ (490, 439), (597, 483), (607, 458), (501, 414)
001- (500, 402), (610, 402), (610, 376), (500, 376)
002- (492, 363), (591, 322), (580, 295), (480, 336)
003- (470, 331), (548, 254), (527, 233), (449, 310)
004- (439, 309), (483, 202), (461, 193), (416, 299)
005- (401, 300), (401, 183), (381, 183), (381, 300)
006- (362, 307), (316, 195), (291, 205), (337, 318)
007- (330, 329), (246, 244), (224, 265), (308, 350)
008- (308, 360), (202, 316), (190, 344), (296, 388)
009- (300, 399), (186, 399), (186, 425), (300, 425)
010- (306, 437), (195, 483), (206, 510), (318, 464)
011- (328, 469), (240, 557), (260, 578), (349, 491)
012- (359, 491), (312, 607), (334, 616), (382, 501)
013- (398, 500), (398, 618), (418, 618), (418, 500)
014- (436, 493), (483, 607), (507, 597), (461, 482)
015- (468, 471), (555, 558), (577, 538), (490, 450)
016- (490, 440), (600, 485), (611, 457), (502, 412)