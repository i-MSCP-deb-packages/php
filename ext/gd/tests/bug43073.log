
---- EXPECTED OUTPUT
(500, 402), (610, 402), (610, 376), (500, 376)
(492, 363), (591, 322), (580, 295), (480, 336)
(470, 331), (548, 254), (527, 233), (449, 310)
(439, 309), (483, 202), (461, 193), (416, 299)
(401, 300), (401, 183), (381, 183), (381, 300)
(362, 307), (316, 195), (291, 205), (337, 318)
(330, 329), (246, 244), (224, 265), (308, 350)
(308, 360), (202, 316), (190, 344), (296, 388)
(300, 399), (186, 399), (186, 425), (300, 425)
(306, 437), (195, 483), (206, 510), (318, 464)
(328, 469), (240, 557), (260, 578), (349, 491)
(359, 491), (312, 607), (334, 616), (382, 501)
(398, 500), (398, 618), (418, 618), (418, 500)
(436, 493), (483, 607), (507, 597), (461, 482)
(468, 471), (555, 558), (577, 538), (490, 450)
(490, 440), (600, 485), (611, 457), (502, 412)
---- ACTUAL OUTPUT
(499, 401), (614, 401), (614, 374), (499, 374)
(492, 363), (598, 319), (588, 294), (481, 338)
(471, 330), (552, 249), (533, 230), (452, 311)
(439, 308), (483, 201), (458, 191), (414, 297)
(401, 300), (401, 185), (374, 185), (374, 300)
(363, 306), (319, 200), (294, 210), (338, 317)
(330, 327), (249, 246), (230, 265), (311, 346)
(308, 359), (201, 315), (191, 340), (297, 384)
(300, 398), (185, 398), (185, 425), (300, 425)
(306, 435), (200, 479), (210, 504), (317, 460)
(327, 468), (246, 549), (265, 568), (346, 487)
(359, 490), (315, 597), (340, 607), (384, 501)
(398, 499), (398, 614), (425, 614), (425, 499)
(435, 492), (479, 598), (504, 588), (460, 481)
(468, 471), (549, 552), (568, 533), (487, 452)
(490, 439), (597, 483), (607, 458), (501, 414)
---- FAILED
