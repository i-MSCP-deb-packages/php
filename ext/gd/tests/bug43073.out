(499, 401), (614, 401), (614, 374), (499, 374)
(492, 363), (598, 319), (588, 294), (481, 338)
(471, 330), (552, 249), (533, 230), (452, 311)
(439, 308), (483, 201), (458, 191), (414, 297)
(401, 300), (401, 185), (374, 185), (374, 300)
(363, 306), (319, 200), (294, 210), (338, 317)
(330, 327), (249, 246), (230, 265), (311, 346)
(308, 359), (201, 315), (191, 340), (297, 384)
(300, 398), (185, 398), (185, 425), (300, 425)
(306, 435), (200, 479), (210, 504), (317, 460)
(327, 468), (246, 549), (265, 568), (346, 487)
(359, 490), (315, 597), (340, 607), (384, 501)
(398, 499), (398, 614), (425, 614), (425, 499)
(435, 492), (479, 598), (504, 588), (460, 481)
(468, 471), (549, 552), (568, 533), (487, 452)
(490, 439), (597, 483), (607, 458), (501, 414)